File: src/FrontEnd/auth-admin/src/types/entities/ids/api-scope-property.ts
interface IApiScopeProperty {
  id?: string;
  key: string;
  value: string;
  scopeId: string;
}

export default class ApiScopeProperty implements IApiScopeProperty {
  id?: string;
  key: string = '';
  value: string = '';
  scopeId: string = '';
}

File: src/Backend/Domain/Auth.Entity/Ids4Entity/ApiScopes/ApiScopeProperty.cs
using System;
using Dapper.Contrib.Plus;

namespace Auth.Entity.Ids4Entity
{
    /// <summary>
    /// 
    /// </summary>
    [Table("api_scope_properties")]
    public class ApiScopeProperty : SeedWork.Entity
    {
        /// <summary>
        /// Key
        /// </summary>
        public string Key { get; set; }

        /// <summary>
        /// Value
        /// </summary>
        public string Value { get; set; }

        /// <summary>
        /// ScopeId
        /// </summary>
        public Guid ScopeId { get; set; }
    }
}
